artemis/db.py

```diff
@@ -28,6 +48,72 @@ def as_dict(self) -> Dict[str, Any]:
             assert False
 
 
+Base = declarative_base()
+engine = create_engine(
+    Config.Data.POSTGRES_CONN_STR, json_serializer=functools.partial(json.dumps, cls=JSONEncoderAdditionalTypes)
+)
+Session = sessionmaker(bind=engine)
+
+
+class TSVector(TypeDecorator):  # type: ignore
+    impl = TSVECTOR
+
+
+class ScheduledTask(Base):  # type: ignore
+    __tablename__ = "scheduled_task"
+    created_at = Column(DateTime, server_default=text("NOW()"))
+    analysis_id = Column(String, primary_key=True)
+    deduplication_data = Column(String, primary_key=True)
+    deduplication_data_original = Column(String)
```

Comment:
why some data is primary_key to table, and why there are two fields related to similar thing?

Reply:
I've added a comment